portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
 <!DOCTYPE html>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<div id="niceback">
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Michael's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<div id="content">
+      <h1><p3 style="color:white">Michael Peters's Portfolio </p3></h1>
+      <center><img src="/images/IMG_3549.jpg" width="120" height="190"></center>
```

Comment:
It's usually a good idea to add an "alt" attribute to images! An alt tag will show the text in the tag. This is good because sometimes the user may have a slow connection and can't load the image, and it's important to accessibility as well for users who have to use screenreaders. 

https://www.w3schools.com/tags/att_img_alt.asp

Reply:
Was not aware of that thank you!